src/style.css

```diff
@@ -0,0 +1,85 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
+
+html {
+  font-family: Avenir, sans-serif;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.form-search {
+  margin-top: 20px;
+}
+
+.input {
+  display: block;
+  width: 100%;
+  color: rgb(61, 78, 97);
+  border: 1px solid rgb(225, 231, 237);
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  outline: none;
+  font-family: inherit;
+  font-weight: 900;
+  text-shadow: 0 4px 4px rgba(0, 0, 0, 0.25);
+  background-image: url(images/Search.svg);
+  background-repeat: no-repeat;
+}
+
+.input-search-big {
+  height: 70px;
+  padding-left: 62px;
+  font-size: 16px;
+  line-height: 22px;
+  background-size: 19px;
+  background-position: 26px 25px;
+}
```

Comment:
input-search-big is redundant class, move all this styles under general input class and rewrite them for a small input